### PR TITLE
freexl: update to 1.0.6.

### PR DESCRIPTION
**Pull request by fosslinux**

### CODE REVIEW

File: srcpkgs/freexl/template [hunk 1]
```diff
@@ -1,14 +1,14 @@
...
 short_desc="Library to extract valid data from within an Excel (.xls) spreadsheet"
 maintainer="Giuseppe Fierro <gspe@ae-design.ws>"
 license="MPL-1.1,GPL-2,LGPL-2.1"
 homepage="https://www.gaia-gis.it/fossil/freexl/index"
 distfiles="http://www.gaia-gis.it/gaia-sins/${pkgname}-${version}.tar.gz"
-checksum=3dc9b150d218b0e280a3d6a41d93c1e45f4d7155829d75f1e5bf3e0b0de6750d
+checksum=3de8b57a3d130cb2881ea52d3aa9ce1feedb1b57b7daa4eb37f751404f90fc22
 
 freexl-devel_package() {
 	depends="${makedepends} ${sourcepkg}-${version}_${revision}"
```
Comment (Piraty) [inline srcpkgs/freexl/template:14]: `makedepends` is not set??

File: srcpkgs/freexl/template [hunk 2]
```diff
@@ -1,14 +1,14 @@
 # Template file for 'freexl'
 pkgname=freexl
-version=1.0.5
+version=1.0.6
 revision=1
 build_style=gnu-configure
 short_desc="Library to extract valid data from within an Excel (.xls) spreadsheet"
 maintainer="Giuseppe Fierro <gspe@ae-design.ws>"
 license="MPL-1.1,GPL-2,LGPL-2.1"
```
Comment (Piraty) [inline srcpkgs/freexl/template:8]: fix license, according to xlint

**Comment by fosslinux** [comment]
Addressed.

File: srcpkgs/freexl/template [hunk 3]
```diff
@@ -1,17 +1,17 @@
...
-license="MPL-1.1,GPL-2,LGPL-2.1"
+license="MPL-1.1,GPL-2-or-later,LGPL-2.1-or-later"
 homepage="https://www.gaia-gis.it/fossil/freexl/index"
 distfiles="http://www.gaia-gis.it/gaia-sins/${pkgname}-${version}.tar.gz"
-checksum=3dc9b150d218b0e280a3d6a41d93c1e45f4d7155829d75f1e5bf3e0b0de6750d
+checksum=3de8b57a3d130cb2881ea52d3aa9ce1feedb1b57b7daa4eb37f751404f90fc22
 
 freexl-devel_package() {
-	depends="${makedepends} ${sourcepkg}-${version}_${revision}"
+	depends="${sourcepkg}-${version}_${revision}"
```
Comment (ericonr) [inline srcpkgs/freexl/template:14]: Shouldn't it be `>=` ?

Comment (fosslinux) [reply srcpkgs/freexl/template:14]: shlib bumps would make `>=` wrong, I think. Not completely sure though

Comment (Piraty) [reply srcpkgs/freexl/template:14]: Manual.md says `>=`

Comment (ahesford) [reply srcpkgs/freexl/template:14]: In practice it probably doesn't matter much, but `>=` is best practice.

Holding the `-devel` package while still upgrading `${sourcepkg}` will, for example, still allow local static linking against an old version without risking breakage of dependants of `${sourceppkg}`. Sure, this is a contrived example, but why prohibit this behavior for no real benefit?

Comment (fosslinux) [reply srcpkgs/freexl/template:14]: Ok, fair points, I get it

Comment (fosslinux) [reply srcpkgs/freexl/template]: Done

**Pull request merged**